### DISCUSSION TITLE
Thanks for the great work

### DISCUSSION THREAD

**Discussion by arussbach** [chuangcaleb/obsidian-fountain-editor#27] [General]
I have been writing my second feature film for years now on final draft, and every day I spent on it was more painful than the last (the bugs, crashes and all the clunky side features like beatboards and Index cards that don't scale well with big projects...) I just now discoverd Obsidian with all it's linking capacity, and it felt like a scriptwriter/director's dream tool and when I discoverd you made this plugin, and made it possible to write the entire scipt in obsidian, I migrated my entire project. I'm only slightly exagerating when I say that this workflow has made me fall in love with my work again, every thing I write, every info I add, seem to integrate organicaly in the project thanks to the linking features of Obsidian. So I just wanted to thank the creater of this plugin for putting in the effort. I beleve Obsidian combined with fountain might be an absolutely amazing tool for scriptwriting and I hope I will also be able to contibute at some stage ! So, thank you.

**Comment by chuangcaleb** [comment]
![Happy Catto Cats GIF](https://media.tenor.com/JWFEQcWcJyQAAAAC/happy-catto-cats.gif)

So glad to hear in detail how this project has helped you! Really made my day.

Please feel free to reach out and ask about anything! If you just only discovered Obsidian, you can try looking in their [community Discord](https://discord.gg/obsidianmd) — I've found a bunch of help and surprising tips on there!

**Comment by arussbach** [comment]
Hey as I told you, I absolutely love using obsidian for screen writing and your plugin is what makes it possible (thx again). And my hobby aside from my directing and screen writing job is actually programming. I have been doing it for 10 years now on the evenings and WEs. I'm fluent in C# F# and Rust, I actually did a year in computer sciences half a decade ago. So obvusly I was wondering if I could help you with improving the plugin. I never worked on collective projects, and I don't know Js, so I'm not sure if I would be a liability or an asset, but if you want to talk to me, don't hesitate, I'would be very happy to contribute how I can. I sent you an invite on discord.

**Reply by chuangcaleb** [reply]
Yeah, I would love to be in contact with you! This is my first open source collective project as well. The stack is Typescript/Javascript + CSS, feel free to pick up a ticket and try to tackle it, 

Or even with your incompatible tech stack, you can also add comments to weigh in your opinions or counterpropose ideas/current implementations!

AND even if you're non-technical, you can also suggest design or user experience improvements! Report bugs! Showcase your work! 

Added you on Discord!
